e2e/test/scenarios/embedding/embedding-native.cy.spec.js

```diff
@@ -320,6 +325,58 @@ describe("scenarios > embedding > native questions", () => {
       });
     });
   });
+
+  describe("locked parameters", () => {
+    beforeEach(() => {
+      const nameParameter = questionDetails2.native["template-tags"]["name"];
+      const sourceParameter =
+        questionDetails2.native["template-tags"]["source"];
+
+      createNativeQuestion(questionDetails2, {
+        wrapId: true,
+      });
+
+      cy.get("@questionId").then(questionId => {
+        cy.request("PUT", `/api/card/${questionId}`, {
+          enable_embedding: true,
+          embedding_params: {
+            [nameParameter.name]: "enabled",
+            [sourceParameter.name]: "locked",
+          },
+        });
+      });
+    });
+
+    it("locked parameters require a value to be specified in the JWT", () => {
```

Comment:
This test is moved from below, and use a different question details because `questionDetailsWithDefaults` that we used previously result in the question result `0`.

By using this new question details, the result is `2500` which looks better.

The rest of the setup is the same.